src/web/lib/getCommentCount.ts

```diff
@@ -0,0 +1,27 @@
+import { joinUrl } from '@root/src/web/lib/joinUrl';
+
+export const getCommentCount = async (ajaxUrl: string, shortUrl: string) => {
+    const url = joinUrl([
+        ajaxUrl,
+        `discussion/comment-counts.json?shortUrls=${shortUrl}`,
+    ]);
+    return fetch(url)
+        .then(response => {
+            if (!response.ok) {
+                throw Error(response.statusText);
+            }
+            return response;
+        })
+        .then(response => response.json())
+        .then(
+            json =>
```

Comment:
can we not use typescript `json?.counts?[0] || 0` ?

Reply:
I wish but we still get linting snags around this